src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,126 @@
+      "description": "@Setting_FormattingOnTypeDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
+        "pass":{
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\FormatOnType"
+          }
+        }
+      }
+    },
+    "textEditor.razor.advanced.autoClosingTags": {
+      "type": "boolean",
+      "default": true,
+      "title": "@Setting_AutoClosingTagsDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_AutoClosingTagsDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
+        "pass":{
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\AutoClosingTags"
+          }
+        }
+      }
+    },
+    "textEditor.razor.advanced.autoInsertAttributeQuotes": {
+      "type": "boolean",
+      "default": true,
+      "title": "@Setting_AutoInsertAttributeQuotesDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_AutoInsertAttributeQuotesDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
+        "pass":{
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\AutoInsertAttributeQuotes"
+          }
+        }
+      }
+    },
+    "textEditor.razor.advanced.colorBackground": {
+      "type": "boolean",
+      "default": true,
+      "title": "@Setting_ColorBackgroundDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_ColorBackgroundDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
+        "pass":{
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\ColorBackground"
+          }
+        }
+      }
+    },
+    "textEditor.razor.advanced.commitElementsWithSpace": {
+      "type": "boolean",
+      "default": true,
+      "title": "@Setting_CommitElementsWithSpaceDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_CommitElementsWithSpaceDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
+        "pass":{
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\CommitElementsWithSpace"
+          }
+        }
+      }
+    },
+    "textEditor.razor.advanced.snippets": {
+      "type": "string",
+      "default": "all",
+      "enum": [
+        "all",
+        "custom",
+        "none"
+      ],
+      "enumItemLabels": [
+        "@Setting_SnippetsAll;{13b72f58-279e-49e0-a56d-296be02f0805}",
+        "@Setting_SnippetsCustom;{13b72f58-279e-49e0-a56d-296be02f0805}",
+        "@Setting_SnippetsNone;{13b72f58-279e-49e0-a56d-296be02f0805}"
+      ],
+      "title": "@Setting_SnippetsDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_SnippetsDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
```

Comment:
Display name and description reversed?

Reply:
Yup, I have it fixed locally. Testing is a bit painful right now so I didn't test before pushing